RELEASE.md

```diff
@@ -1,30 +1,34 @@
 # Release Instructions/Checklist
 
-This file describes the required steps to release neow3j.
-Before releasing make sure that the external documentation at [neow3j.io](https://neow3j.io) is up 
-to date.
+This file describes the required steps to release neow3j. Before releasing make sure that the external documentation
+at [neow3j.io](https://neow3j.io) is up-to-date.
 
 ## Release the Source Code
 
-- Bump the version on the develop branch (e.g., `develop-3.x`). Do a global search with the current 
-  version number and replace it with the new version number. There should be four affected 
-  locations in the `README.md`, one in the `build.gradle`, and one in `Compiler.java`.
-  - The tests NNSIntegrationTest, NFTIntegrationTest, DivisibleNFTIntegrationTest, 
-    and NonDivisibleNFTIntegrationTest will fail because of the version change in the compiler.
-   
-- Create a Pull Request from develop to master (e.g., from `develop-3.x` to `master-3.x`) -- this
-is called a "Release Pull Request"
-  - Set the name as "Release x.x.x"
-  - Set the correct milestone, project and a reviewer
-   
-- Review and merge the Release Pull Request, mainly checking for critical changes **or**
-changes that shouldn't be in the release
- 
-- Tag the main branch at the commit to be release (e.g., `3.2.1`)
- 
-- Right after merging, bump the version in the `build.gradle` file (`develop-3.x` branch)
+- Create a branch `release` from `main`.
+- Bump the neow3j version on that branch to the release version. Do a global search with the previous version number and
+  replace it with the new version number. There should be two affected files, i.e., `README.md`, and `DEVELOPMENT.md`.
+- Verify that the release version is also set correctly in the files `build.gradle` and `Compiler.java`.
+    - In case that the version number in `build.gradle` and `Compiler.java` need updating, the tests
+      `NNSIntegrationTest`, `NFTIntegrationTest`, `DivisibleNFTIntegrationTest`, and
+      `NonDivisibleNFTIntegrationTest` will fail because of the version change in the compiler.
+        - Change the contract hashes in these tests accordingly to make them pass again.
+
+- Create a Pull Request from `release` to `main` -- this is called a "Release Pull Request".
+    - Set the name as "Release x.x.x".
+    - Set the correct milestone, project and a reviewer.
+
+- Review the Release Pull Request, mainly checking for critical changes **or** changes that shouldn't be in the release.
+- Merge the Pull Request, and then, delete the `release` branch.
+
+- Tag the `main` branch at the commit to be release (e.g., `3.14.0`).
+
+- Right after merging, bump the version in the `build.gradle` and `Compiler.java` file on the `main` branch. Thus, also
```

Comment:
Oh, here it goes. Why do we need to bump on earlier steps as well?!?

Reply:
See my comment above.